### ISSUE TITLE
RPM-Building on Centos7 fails

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#444]
`rpmbuild -ta nx-libs-3.5.99-6.tar.gz` fails with mesa problems because the quilt call is missing...

```
./mesa-quilt push -a
./mesa-quilt: line 32: quilt: command not found
```

RHEL7/CentOS7 does not provide `quilt` out of the box, you need to take the one from EPEL. We had lots of problems at our customers when running EPEL software (mainly because of unmaintained dependencies), so it is generally not a good idea to depend on such packages. As I am experiencing other inconveniences with this mesa-quilt script when doing development I think we should find another approach for this.

**Comment by sunweaver** [comment]
The only approach I see then his having a fallback in "mesa-quilt" that simply uses the patch command.

**Comment by uli42** [comment]
Yea, something like that. As there's no huge patch collection
currently we could drop quilt completely. Not sure if the number will
grow large, though...

**Comment by uli42** [comment]
Also, the quilt dependency is missing in the spec file.

**Comment by uli42** [comment]
Maybe git-series could be used: https://github.com/git-series/git-series

> git-series tracks both a patch series and its evolution within the same git repository. git-series works entirely with existing git features, allowing git to push and pull a series to any git repository along with other branches and tags.

**Comment by sunweaver** [comment]
From closed/duplicate bug #89... We need to make sure that such keystrokes also work in shadowed sessions.

From X2Go BTS:
http://bugs.x2go.org/cgi-bin/bugreport.cgi?bug=846
package: nxagent

Currently, there doesn't seem to be a way to cleanly terminate a
shadow session running in fullscreen mode.

Ctrl-Alt-T doesn't work, Alt-Tab gets sent to the remote end, so one
can only run x2goterminatesession on the remote end or kill the
x2goclient task on the local side.
